starter-code/functions-and-arrays.js

```diff
@@ -82,9 +116,16 @@ var words = [
 var uniqued = uniquifyArray(words);
 console.log(uniqued);
 
+
+
 // Finding Elements
 function doesWordExist (wordsArray, word) {
-
+  for (var i = 0; i < wordsArray.length; i++) {
+  if (wordsArray[i] === word) {
```

Comment:
Be careful with the indentation. You should use an editor that fixes this for you.

Reply:
Would it be possible to solve this one with a `forEach`? I'm not sure how that'd work with the `wordsArray[i]` being used here.

Also, would using `array.includes(word)`

Reply:
`wordsArray.includes(word)`shoud work. Did you return it?

For the `forEach`solution, please post your attempt here as a comment and I will guide you toward the solution if you didn't succeed.